project/models.py

```diff
@@ -42,7 +42,19 @@ def __str__(self):
         return self.name
 
 class Group(VersionedModel):
-    name = models.CharField(max_length=100)
+    image                   = FilerImageField(null=True, blank=True, on_delete=models.SET_NULL, related_name='group_image')
+    name                    = models.CharField(max_length=100)
```

Comment:
Can we add a slug field and use that in the URI rather than ID? Nicer to read and better for SEO.